std_msgs/Header header
string obj_name
bool is_exact
---
uint8 SUCCEEDED=0
uint8 FAILED=1
uint8 result_status